runelite-client/src/main/java/net/runelite/client/plugins/runecraft/RunecraftPlugin.java

```diff
@@ -97,6 +107,9 @@
 	@Inject
 	private AbyssOverlay abyssOverlay;
 
+	@Inject
+	private ZeahOverlay zeahOverlay;
```

Comment:
Same as above, please use proper indentation.

Reply:
Also, maybe `DenseEssenceOverlay` would be more correct/clear about what this shows?